src/main/java/com/jagrosh/giveawaybot/Bot.java

```diff
@@ -107,11 +108,15 @@ public boolean startGiveaway(TextChannel channel, User creator, Instant now, int
             return false;
         database.settings.updateColor(channel.getGuild());
         Instant end = now.plusSeconds(seconds);
+        String emoji = database.settings.getSettings(channel.getGuild().getIdLong()).getEmojiDisplay();
         Message msg = new Giveaway(0, channel.getIdLong(), channel.getGuild().getIdLong(), creator.getIdLong(), end, winners, prize, Status.RUN, false)
                 .render(channel.getGuild().getSelfMember().getColor(), now);
-        channel.sendMessage(msg).queue(m -> 
-        {
-            m.addReaction(Constants.TADA).queue();
+
+        channel.sendMessage(msg).queue(m -> {
+            m.addReaction(emoji).onErrorFlatMap(ignored -> { // this might be perms error or because we can't add that emoji
```

Comment:
It might be a good idea to tell the user that the custom emoji can't be found anymore?